openstack/provision.sh

```diff
@@ -5,10 +5,10 @@ SCRIPT_DIR="$(cd "$(dirname "$0")" && pwd)"
 source "$SCRIPT_DIR/common.sh"
 
 echo "Provisioning $EIRINI_STATION_USERNAME-eirini-station"
-scp -r ~/workspace/eirini-station/provision*.sh "$EIRINI_STATION_USERNAME@$STATION_IP:/tmp"
-ssh -A "$EIRINI_STATION_USERNAME@$STATION_IP" "sudo /tmp/provision.sh"
-ssh -A "$EIRINI_STATION_USERNAME@$STATION_IP" "/tmp/provision-user.sh"
+scp -r ~/git/eirini-station/provision*.sh "$VMUSER@$STATION_IP:/tmp" 
```

Comment:
It is a team convention to clone all git repos in `~/workspace` and all our scripts assume that path. Can you revert that path please?

Reply:
sure, please excuse